libs/common/src/constants/facility-type-mapped.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * These are types featured in the mapped facility snapshot in both 'mapGefFacility.js' and 'mapFacility.js',
+ * in the file 'mapFacilityType.js'.
+ *
+ * They are slightly different from the usual facility types and are tech debt.
+ *
+ * These are subtly different from how we format types for user facing areas (ie bond),
+ * and if making user facing facility types, a different constant should be used
+ */
+export const MAPPED_FACILITY_TYPE = {
+  // Bond is derived from bondType in these mappers and is deliberately omitted here
+  // Loan is identical to facility type, but is provided to avoid confusion
+  // if types are ever introduced fully and driven from the constant
+  LOAN: 'Loan',
+  CASH: 'Cash facility',
+  CONTINGENT: 'Contingent facility',
+} as const;
```

Comment:
This is due to some idiosyncrasies with this field. The existing implimentation does not line up with how it is displayed in the UI, and does not line up with the backend. It's true use is rather unknown. It was originally constructed through string joining of another unconnected constant -- which was incorrect so I'm pulling it out here.

I'm going to update the comment a bit more, but the tldr is the old implimentation is not correct, it should be driven by its own constant and not relying on being string joined

Reply:
(also anyone deep enough into the code to be worrying about this will have enough context to understand I hope, as this is also only used to modify a snapshot, which I would have thought should be immutable)

Reply:
Thanks @AlexBramhill I am still not fully understanding though and why we need to introduce this as part of this PR? Maybe can have a quick chat about it - just concerned might be potentially introducing tech debt here?

What was the old implementation?

Should we add a TODO and/or ticket number to the "this is tech debt" part of the documentation?

@Zainzzkk What do you think?

Reply:
Lets have a chat tomorrow @ttbarnes -- I hope this should be reducing tech debt rather than incurring it

Reply:
Thanks @AlexBramhill nice changes